apps/api/pecanapi-spec.yml

```diff
@@ -992,6 +1028,123 @@ paths:
           description: Access forbidden
         '404':
           description: Run data not found
+          
+  /api/posteriors/:
+    get:
+      tags:
+        - posteriors
+      summary: Search for the posteriors
+      parameters:
+        - in: query
+          name: pft_id
+          description: If provided, returns all posteriors for the provided model_id
+          required: false
+          schema:
+            type: string
+        - in: query
+          name: host_id
+          description: If provided, returns all posteriors for the provided host_id
+          required: false
+          schema:
+            type: string
+        - in: query
+          name: offset
+          description: The number of posteriors to skip before starting to collect the result set.
+          schema:
+            type: integer
+            minimum: 0
+            default: 0
+          required: false
+        - in: query
+          name: limit
+          description: The number of posteriors to return.
+          schema:
+            type: integer
+            default: 50
+            enum:
+              - 10
+              - 20
+              - 50
+              - 100
+              - 500
+          required: false
+      responses:
+        '200':
+          description: List of posteriors
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  inputs:
+                    type: array
+                    items: 
+                      type: object
+                      properties:
+                        id:
+                          type: string
+                        file_name:
+                          type: string
+                        file_path:
+                          type: string
+                        pft_name:
+                          type: string
+                        tag:
+                          type: string
+                        hostname:
+                          type: string
+                        start_date:
+                          type: string
+                        end_date:
+                          type: string
+                  count:
+                    type: integer
+                  next_page:
+                    type: string
+                  prev_page:
+                    type: string
+                
+        '401':
+          description: Authentication required
+        '403':
+          description: Access forbidden
+        '404':
+          description: Workflows not found
```

Comment:
isn't it the posterior, not the workflow, that wasn't found?

Reply:
This API is to download multiple files from a workflow!